pkg/controller/flowsconfig/flowsconfig_controller.go

```diff
@@ -0,0 +1,135 @@
+package flowsconfig
+
+import (
+	"context"
+	"log"
+	"strconv"
+	"time"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/openshift/cluster-network-operator/pkg/names"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+const (
+	ovsFlowsConfigMapName   = "ovs-flows-config"
+	ovsFlowsConfigNamespace = names.APPLIED_NAMESPACE
+)
+
+type FlowsConfig struct {
+	// Target IP:port of the flow collector
+	Target string
+
+	// CacheActiveTimeout is the max period during which the reporter will aggregate flows before sending
+	CacheActiveTimeout *uint
```

Comment:
The pointers are used to detect if a feature is enabled or not, as they are optional configuration values. Another options could be:
- using the zero value (if `0 => disabled`) but it might be not so clear for some properties where zero is a possible value.
- adding some kind of extra boolean property, like a boolean saying if it is enabled or not, but it could maybe dirty the code.

Reply:
Its fine. If anyone can think of a cleaner idea, I am all ears but its fine as it is.